frontend/src/views/InvoiceView.vue

```diff
@@ -109,128 +133,104 @@ onMounted(async () => {
-    </router-link>
-    <div class="flex gap-x-2">
-      <Button label="Speichern" text raised />
-      <Button label="Löschen" severity="danger" text raised />
-    </div>
-  </div>
-  <Card class="my-2">
-    <template #content>
-      <div class="mb-4 font-bold">Auftragsdaten</div>
-      <div><span class="font-bold"> Bauvorhaben: </span>{{ orderInfo?.title }}</div>
-      <div>
-        <span class="font-bold"> Auftrags-Nr.: </span>
-        {{ orderInfo?.id }}
-      </div>
-      <div><span class="font-bold">Status: </span> {{ orderInfo?.status }}</div>
-      <div>
-        <span class="font-bold">Kunde: </span>
-        <router-link class="underline" :to="`${ROUTES.CLIENT.path}/${orderInfo?.client_id}/edit`">
-          {{ orderInfo?.client.first_name }} {{ orderInfo?.client.last_name }}
-          <i class="pi pi-external-link ml-1"></i>
-        </router-link>
+  <div v-if="orderInfo">
+    <div class="flex flex-row justify-between">
+      <router-link :to="`${ROUTES.ORDER.path}/${route.params.order_id}/edit`">
+        <Button icon="pi pi-arrow-left" size="small" severity="secondary" text raised />
+      </router-link>
+      <div class="flex gap-x-2">
+        <Button @click="onCreateInvoice" label="Speichern" text raised />
+        <!-- TODO: add removal -->
+        <!-- <Button label="Löschen" severity="danger" text raised /> -->
       </div>
-      <div>
-        <span class="font-bold">Skonto: </span>
-        <span>{{ orderInfo?.can_have_cash_discount ? "Ja" : "Nein" }}</span>
-      </div>
-    </template>
-  </Card>
-  <Card class="my-2">
-    <template #content>
-      <div class="mb-4 font-bold">Rechnung:</div>
-      <FloatLabel class="my-6">
-        <Calendar
-          id="invoice-date-input"
-          v-model="invoiceDate"
-          dateFormat="dd/mm/yy"
-          showIcon
-          iconDisplay="input"
-        />
-        <label for="calendar"> Rechnungsdatum </label>
-      </FloatLabel>
-      <FloatLabel class="my-6">
-        <Calendar
-          id="payment-target-input"
-          v-model="paymentTarget"
-          dateFormat="dd/mm/yy"
-          showIcon
-          iconDisplay="input"
-        />
-        <label for="calendar"> Zalungsziel </label>
-      </FloatLabel>
-      <div class="mb-2">Zahlungstatus:</div>
-      <Dropdown
-        v-model="invoiceInfo.status"
-        :options="invoiceType"
-        placeholder="Anrede"
-        class="w-full md:w-[14rem] mb-3"
-      />
-      <div class="flex flex-row justify-between items-center mb-4">
-        <div class="font-bold">Leistungsdatum:</div>
-        <Button @click="onServiceDateCreate" icon="pi pi-plus" rounded outlined />
-      </div>
-      <div
-        class="flex flex-row justify-between items-center"
-        v-for="(item, idx) in serviceDates"
-        :key="item.id"
-      >
+    </div>
+    <OrderSummary :order-info="orderInfo" />
+    <Card class="my-2">
+      <template #content>
```

Comment:
A lot of space is used here on the right side of those small inputs.
Why not use something that uses the full width and can wrap around?

<img width="937" alt="image" src="https://github.com/user-attachments/assets/214f48db-a12c-4f7e-81df-d1e4c994f8c5">

Reply:
<img width="1558" alt="image" src="https://github.com/user-attachments/assets/134b315c-0644-48cf-8823-6ad49e779a21">